diff --git a/source/gameshared/q_arch.h b/source/gameshared/q_arch.h
index 26f1557a..5554efcc 100644
--- a/source/gameshared/q_arch.h
+++ b/source/gameshared/q_arch.h
@@ -202,7 +202,7 @@ typedef UINT_PTR socket_handle_t;
 #define OPENAL_RUNTIME
 
 // FIXME: move these to CMakeLists.txt
-#define LIBZ_LIBNAME "libz.so.1|libz.so"
+#define LIBZ_LIBNAME "libz.so"
 #define LIBCURL_LIBNAME "libcurl.so.4|libcurl.so.3|libcurl.so"
 #define LIBPNG_LIBNAME "libpng16.so.16|libpng15.so.15|libpng14.so.14|libpng12.so.0|libpng.so"
 #define LIBJPEG_LIBNAME "libjpeg.so.8|libjpeg.so"
